### DISCUSSION TITLE
Standup (Wednesday, May 5, 2022)

### DISCUSSION THREAD

**Discussion by CarlosCRG19** [CarlosCRG19/ijalti-peers#42] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on Figma wireframes
 - Gave tech talk on Git and GitHub

**What are your priorities for the next 24 hours?**:
 - Finish Figma wireframes

**Blockers**:
 - Figma UI x2

**Shoutouts**:
 - @everyone cause you listened to the whole tech talk

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Listen to a cracked guy talk about Github

**What are your priorities for the next 24 hours?**:
 - Material UI
 - Clone the developer branch
 - Start a client branch with the basic stuff

**Blockers**:
 - None

**Shoutouts**:
- @CarlosCRG19 for his dedication and GitHub knowledge

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Created our app's first endpoint!

**What are your priorities for the next 24 hours?**:
 - Read TypeORM docs

**Blockers**:
 - Still, poor knowledge on TypeORM

**Shoutouts**:
 - @CarlosCRG19 for the Git talk yesterday!

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Define first typeorm entities and start working on first endpoints

**What are your priorities for the next 24 hours?**:
 - Create connection between backend and database 

**Blockers**:
 - Adapting to TypeOrm

**Shoutouts**:
 - @CarlosCRG19  for yesterday's git crash course
 - @antonieto  for helping me getting started with typeScript

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Read about Material UI 
 - Started working on a "create job offer" view

**What are your priorities for the next 24 hours?**:
 - Keep working on the view

**Blockers**:
 - Material UI is not as easy as I thought 

**Shoutouts**:
 - @CarlosCRG19 for the great talk
